ansible/roles/test/tasks/pfc_wd/functional_test/functional_test.yml

```diff
@@ -25,13 +27,30 @@
       pfc_wd_storm_template: pfc_storm_mlnx.j2
   when: peer_hwsku == "MLNX-OS"
 
+- set_fact:
+      pfc_wd_storm_template: pfc_storm_arista.j2
+      pfc_wd_storm_stop_template: pfc_storm_stop_arista.j2
+  when: peer_hwsku | search("Arista") or peer_hwsku | search("arista")
+
 - name: Set timers
   set_fact:
       pfc_wd_detect_time: 200
       pfc_wd_restore_time: 200
       pfc_wd_restore_time_large: 30000
       pfc_wd_poll_time: 100
 
+- name: Create pfc generator script in case it doesn't exist
+  file: path=/mnt/flash/pfctest.py state=touch
```

Comment:
why touch?

Reply:
Copy the script will fail if the file doesn't exist on this platform. So create the file first.